pkg/controller/direct/gkehub/featuremembership_controller.go

```diff
@@ -277,3 +285,22 @@ func (a *gkeHubAdapter) Export(context.Context) (*unstructured.Unstructured, err
 	u.Object = uObj
 	return u, nil
 }
+
+// mID is in the format of "projects/{ProjectID}/locations/*/memberships/{membershipId}".
+// canonicalizeMembershipID cononicalizes it to "projects/{ProjectNumber}/locations/*/memberships/{membershipId}".
+func canonicalizeMembershipID(mID string, feature *featureapi.Feature) (string, bool, error) {
```

Comment:
let's have this renamed to something like matchWithCanonicalMemebrshipID
```suggestion
// mID is in the format of "projects/{ProjectID}/locations/*/memberships/{membershipId}".
// matchWithCanonicalMemebrshipID matches the membershipID to the canonical membershipID "projects/{ProjectNumber}/locations/*/memberships/{membershipId}".
func matchWithCanonicalMemebrshipID(mID string, feature *featureapi.Feature) (string, bool, error) {
```

Reply:
done